.travis.yml

```diff
@@ -3,15 +3,20 @@ os:
   - linux
   - osx
 node_js:
-  - "8"
-before_install:
-- npm -g install npm@4
-script: npm run ci
+  - "10"
+script:
+  - node --version
+  - npm --version
+  - npm run ci
 notifications:
   email: false
 deploy:
   - provider: script
-    script: npm run release
+    script:
+      - node --version
+      - npm --version
+      - npm run build:installer
+      - npm run release
```

Comment:
* `npm run build:installer` generates the native installer for the current platform (in the case of a Travis macOS VM, a macOS `'.pkg'` installer is created; in the case of an Appveyor Windows VM, a Windows `'.exe'` installer is created).
* `npm run release` creates or updates a GitHub release, uploading the installer files.